sql/src/main/java/org/apache/druid/sql/calcite/rel/PartialDruidQuery.java

```diff
@@ -388,16 +388,14 @@ public boolean canAccept(final Stage stage)
     } else if (stage.compareTo(currentStage) <= 0) {
       // Cannot go backwards.
       return false;
-    } else if (stage.compareTo(Stage.AGGREGATE) > 0 && stage.compareTo(Stage.SORT) < 0 && aggregate == null) {
-      // Cannot do post-aggregation stages without an aggregation.
-      return false;
-    } else if (stage.compareTo(Stage.SORT) > 0 && sort == null) {
-      // Cannot do post-sort stages without a sort.
-      return false;
-    } else {
+    } else // Cannot do post-sort stages without a sort.
```

Comment:
Something is weird with the comments here… automated rewrite gone wrong?

Reply:
YEah, looks like it re-wrote the final one to actually just return instead of be a branch and then had a hard time with the comments...